### DISCUSSION TITLE
When working with source, does Ctrl E save the source before compiling?

### DISCUSSION THREAD

**Discussion by dhfreinkel** [codefori/vscode-ibmi#797] [IBM i]
I am using Ctrl S first, but if I do not need to, I would like to know.
Thanks. Darryl.

**Comment by Wright4i** [comment]
That's a setting you can change on your Connection. Click your connection on the bottom bar then check "Auto Save for Actions"
![image](https://user-images.githubusercontent.com/93156666/183156576-536feab7-bf4a-4272-ba1b-2837bc3d9163.png)
